packages/core/src/lib/database/migrations/1744014217496-CreateZapierWebhookSubscriptionRepositoryTable.ts

```
@@ -0,0 +1,177 @@
+import { DatabaseTypeEnum } from '@gauzy/config';
+
+export class CreateZapierWebhookSubscriptionRepositoryTable1744014217496 implements MigrationInterface {
+	name = 'CreateZapierWebhookSubscriptionRepositoryTable1744014217496';
+
+	/**
+	 * Up Migration
+	 *
+	 * @param queryRunner
+	 */
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		console.log(chalk.yellow(this.name + ' start running!'));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)